docs/source/_static/environment.yml

```diff
@@ -0,0 +1,28 @@
+name: apple_tensorflow
+channels:
+  - conda-forge
+  - nodefaults
+dependencies:
+  - absl-py
+  - astunparse
+  - gast
+  - google-pasta
+  - grpcio
+  - h5py
+  - ipython
+  - keras-preprocessing
+  - numpy
+  - opt_einsum
+  - pip=20.2.4
```

Comment:
This seems to be very important that it is == 20.2.4?

Reply:
I would suggest that it was the last version they tested. It should work fine with later versions. I haven't done any testing and did not write it myself. I also am not so interested in running version tests. The full install works with this version though so it is okay for now and we can update it as Apple updates their software.